x-pack/plugin/watcher/src/test/java/org/elasticsearch/xpack/watcher/notification/email/EmailTemplateTests.java

```diff
@@ -88,6 +93,8 @@ public void testEmailTemplateParserSelfGenerated() throws Exception {
         assertThat(emailTemplate.textBody, equalTo(parsedEmailTemplate.textBody));
         assertThat(emailTemplate.htmlBody, equalTo(parsedEmailTemplate.htmlBody));
 
+
+
```

Comment:
remove the extra spaces

Reply:
fixed in https://github.com/elastic/elasticsearch/pull/44460/commits/6261d538c2e4819c452c97193065cd1e2d82bad1